src/Core/src/Platform/Android/ContextExtensions.cs

```diff
@@ -329,8 +329,11 @@ internal static int CreateMeasureSpec(this Context context, double constraint, d
 
 			return mode.MakeMeasureSpec(deviceConstraint);
 		}
-
 		public static float GetDisplayDensity(this Context context) =>
 			context.Resources?.DisplayMetrics?.Density ?? 1.0f;
+		internal static ColorStateList? GetColorListStateForStyleableAttribute (this Context? context, string styleable, int resid, int index)
+		{
+			return PlatformInterop.GetColorStateListForStyleableAttribute (context, styleable, resid, index);
+		}
```

Comment:
Maybe we shouldn't even create a C# method here and just call `PlatformInterop`.

Reply:
I can try that